src/main/java/net/dv8tion/jda/api/managers/PermOverrideManager.java

```diff
@@ -46,11 +46,11 @@
 public interface PermOverrideManager extends Manager<PermOverrideManager>
 {
     /** Used to reset the denied field */
-    long DENIED      = 0x1;
+    long DENIED      = 1;
     /** Used to reset the granted field */
-    long ALLOWED     = 0x2;
+    long ALLOWED     = 1 << 1;
     /** Used to reset <b>all</b> permissions to their original value */
-    long PERMISSIONS = 0x3;
+    long PERMISSIONS = 1 << 2;
```

Comment:
The original value here is 0x3, not 0x4 (not sure which is erroneous)

Reply:
0x3 was correct since its supposed to be ALLOWED + DENIED.

```suggestion
    long PERMISSIONS = ALLOWED | DENIED;
```